sdks/nodejs/package-lock.json

```diff
@@ -2159,7 +2159,7 @@
       }
     },
     "y18n": {
-      "version": "4.0.0",
+      "version": "4.0.1",
```

Comment:
I ran `npm audit fix` and the lines below do not correspond to this version so this part of the config is inconsistent.
If you run the same command it will fix up the file.
Also the version of the sdk is inconsistent with package-lock.json so you'll see an update from 1.10.0-dev to 1.14.0-dev too in this file